src/main/java/uk/nhs/hee/tis/trainee/forms/config/StringDeserializer.java

```diff
@@ -0,0 +1,14 @@
+package uk.nhs.hee.tis.trainee.forms.config;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import java.io.IOException;
+
+public class StringDeserializer extends JsonDeserializer<String> {
+
+  @Override
+  public String deserialize(JsonParser p, DeserializationContext ctxt) throws IOException {
+    return p.getValueAsString().trim();
```

Comment:
`String.strip()` covers some edge cases that we might want to cover off while we're at it.